src/CommunityToolkit.Maui/Views/GravatarImageSource.shared.cs

```diff
@@ -0,0 +1,242 @@
+﻿namespace CommunityToolkit.Maui.Views;
```

Comment:
Sorry hopefully the final point, I missed this initially but can we move this out to the root namespace?

Reply:
Sorry I think it means all the sample bits and tests also need to change their namespace

Reply:
I wasn't sure that this should be in the `Views` folder, but I couldn't see anywhere else it should reside.  Especially with the `SectionModel` defining where it belongs also.
Would we need a whole new `Gallery` just for this?

Reply:
Very good point:

Team (@pictos, @VladislavAntonyuk, @brminnick, @jfversluis, @kphillpotts) what do you think to where the `GravatarImageSource` should live? I have the following suggestions please feel free to vote with the appropriate emoji

- 🪟  - `CommunityToolkit.Maui.Views`
- 🌴  - `CommunityToolkit.Maui`
- 📷  - `CommunityToolkit.Maui.ImageSources`

Reply:
It's fine to keep the sample page in `Views`. 

One alternative would be to create a .NET MAUI Community Toolkit Gravatar on Gravatar, then add it to the WelcomePage using `GravatarImageSource`.

Reply:
That would solve where to have it in the sample, and thus eliminate the need for any additional `Gallery`.  However, where would the control exist?  In the root of `Maui` or still within the `Views` folder?

I think that would be a great idea having a MCT gravatar image (and in the MCT we default to that).  However, it needs to be registered against an email address, and then someone is thus responsible for ownership.

Reply:
No worries there! If you're comfortable with it, George, go ahead and make one since you did the work to bring GravatarImageSource to the Toolkit. I think you owning it makes a fitting tribute to your legacy  and the awesome work you've done for the Toolkit 💯 

We can always update the GravatarImageSource in a future PR if you ever need to cancel or replace your Gravatar account.

Reply:
Recognition is not a motivator for me, and I think I'd prefer us to keep the original XFCT author as the default sample Gravatar. 😁

Reply:
I think I'm leaning towards leaving it where it is for now, and if other similar enhancements come along, it can be re-assessed then.

Reply:
I was expecting notifications but I didn't get any on the emoji reactions. It looks like 🎉 got it!

@GeorgeLeithead are you ok with moving the bits over to `CommunityToolkit.Maui.ImageSources`?

Reply:
I have created the folder `ImageSources`, moved the concrete class to `CommunityToolkit.Maui.ImageSources`.  Similarly for the unit tests.
I have also added the XamlNameSpace aliase.
I have however, left the Samples within the `Views` folder and gallery (Rather than having another gallery, specifically for ImageSources - but happy to do so if others think it's needed!).

Reply:
Thanks @GeorgeLeithead. The pedant in me prefers the sample in the same namespace as the source so I have made that change. Hopefully I haven't broken anything as I am failing to build the toolkit on my Mac right now.

Reply:
> Thanks @GeorgeLeithead. The pedant in me prefers the sample in the same namespace as the source so I have made that change. Hopefully I haven't broken anything as I am failing to build the toolkit on my Mac right now.

I'll get this later today and make sure it's all good.

Reply:
All moved and tested.